vars/mavenNode.groovy

```diff
@@ -5,6 +5,19 @@ def call(Map parameters = [:], body) {
     def defaultLabel = buildId('maven')
     def label = parameters.get('label', defaultLabel)
 
+    def file = new File("/var/lib/jenkins/nodes")
```

Comment:
since `'/var/lib/jekins/nodes'`' is repeating did you consider storing that in a variable?